scripts/itest_part.sh

```diff
@@ -5,19 +5,22 @@ WORKDIR=$(pwd)/lntest/itest
 
 TRANCHE=$1
 NUM_TRANCHES=$2
+ITEST_FLAGS=$3
 
-# Shift the passed parameters by two, giving us all remaining testing flags in
+# Shift the passed parameters by three, giving us all remaining testing flags in
 # the $@ special variable.
 shift
 shift
+shift
 
 # Windows insists on having the .exe suffix for an executable, we need to add
 # that here if necessary.
 EXEC="$WORKDIR"/itest.test"$EXEC_SUFFIX"
 LND_EXEC="$WORKDIR"/lnd-itest"$EXEC_SUFFIX"
-echo $EXEC -test.v "$@" -logoutput -goroutinedump -logdir=.logs-tranche$TRANCHE -lndexec=$LND_EXEC -splittranches=$NUM_TRANCHES -runtranche=$TRANCHE
+BTCD_EXEC="$WORKDIR"/btcd-itest"$EXEC_SUFFIX"
```

Comment:
what is the benefit of explicitly specifying it here?

Reply:
You mean why in this file and not in the `Makefile`? I guess I wanted to avoid passing in even more parameters.